api/test/wfl/integration/executor_test.clj

```diff
@@ -24,7 +24,7 @@
   {:type testing-queue-type :queue (ArrayDeque. items)})
 
 (defn ^:private testing-queue-peek [this]
```

Comment:
We repeat these testing queue def'ns in a few different tests, could be candidates to pull out to`wfl.integration.modules.shared`.

Reply:
Yeah seems like a reasonable thing to do. I keep finding bugs and have to fix them all over!